src/javascript/utils.js

```diff
@@ -233,5 +257,7 @@ module.exports = {
 	triggerPage: triggerPage,
 	getValueFromCookie: getValueFromCookie,
 	getValueFromUrl: getValueFromUrl,
-	getValueFromJsVariable: getValueFromJsVariable
+	getValueFromJsVariable: getValueFromJsVariable,
+	sanitise: sanitise,
+	assignIfUndefined: assignIfUndefined
```

Comment:
Looks like we have a mix of trailing commas and no trailing commas (i.e. on `src/libs/get-trace.js`)

If it this is true for the rest of the repo perhaps it could be cleaned up in a separate PR.